app/models/solr_document.rb

```diff
@@ -40,4 +40,17 @@ def thumbnail_reference?
   def carto_reference
     nil
   end
+
+  def sidecar
```

Comment:
I wonder if this should be memo-ized? I don't know how often it's accessed, but this would re-save every time it was touched.

Reply:
Memoizing doesn't help here, BUT the sidecar shouldn't be saved with every page load. Fixed in newest commit.